integration/gethnetwork/geth_network.go

```diff
@@ -353,3 +356,39 @@ func waitForIPC(dataDir string) {
 		counter++
 	}
 }
+
+func EnsurePortsAreAvailable(startPort int, numberNodes int) error {
```

Comment:
@BedrockSquirrel I feel like this should be called as part of `NewGethNetwork`, instead of being a public function. In what cases would you not want the ports to be available?

Reply:
@joeldudley Yeah that's fair, will change. I was following the pattern of EnsureBinariesExist() but hadn't clocked that that actually provides the path for the wiring too.